src/main/java/com/cloudbees/jenkins/support/configfiles/SecretHandler.java

```diff
@@ -50,8 +49,8 @@ class SecretHandler {
      * FALLBACK will be disable in case you define a system property like -Dsupport-core-plugin.SecretHandler.ENABLE_FALLBACK=false
      * Otherwise will be enabled.
      */
-    private static boolean ENABLE_FALLBACK = !StringUtils.equalsIgnoreCase(
-            System.getProperty("support-core-plugin.SecretHandler.ENABLE_FALLBACK", "TRUE"), "FALSE");
+    private static boolean ENABLE_FALLBACK =
+            !"FALSE".equalsIgnoreCase(System.getProperty("support-core-plugin.SecretHandler.ENABLE_FALLBACK", "TRUE"));
```

Comment:
BTW this is done more easily with `Boolean.getProperty` or `SystemProperties.getBoolean`.